src/site/twitch.tv/modules/hidden-elements/HiddenElementsModule.vue

```diff
@@ -216,6 +222,12 @@ export const config = [
 	}
 }
 
+.seventv-hide-featured-clips {
+	div[class="ScTransitionBase-sc-hx4quq-0 gaROJz tw-transition"] div[class="Layout-sc-1xcs6mc-0 hslCHI"] {
+		display: none !important;
+	}
+}
+
```

Comment:
Layout classes are autogenerated every time twitch rebuilds its website and these would quickly become invalid